.github/workflows/ci.yml

```diff
@@ -95,9 +97,15 @@ jobs:
         name: Build snps2vcf Docker Image
         with:
           context: .
-          push: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
```

Comment:
Should this condition remain, since the image should only be built once merged to main?

Reply:
Apologies, still testing - would add the condition back before merge